src/components/button/README.md

```diff
@@ -19,11 +19,8 @@ interface Props {
 ## Button types
 
 ```typescript
-export enum ButtonType {
-  default = "default",
-  noFill = "noFill",
-  borderless = "borderless",
-}
+  type ButtonType = "default" | "noFill" | "borderless",
```

Comment:
Nope. Why is this required to change?

Reply:
As I understood, the inconvenience is that you have to import enum from the component to set the type other than default. Correct me if I'm wrong. If so, I'm 100% for this change, you should not be obliged to import enum types from UI component module to be able to set some customization.

Reply:
Why should I do
```typescript
import {Button, ButtonTypes } from '@netdata/netdat-ui'
function Component() {
  return(<>
    <Button label="any" type={ButtonType.noFill} .../>
  </>)
}
```
Instead of just "noFill". A lot of redundant extra complexity